netket/hilbert/continuous_particle.py

```diff
@@ -0,0 +1,69 @@
+from typing import Tuple, Union, AnyStr
+
+from .abstract_continuous_space_particle import ContinuousHilbert
+
+
+class ContinuousParticle(ContinuousHilbert):
+    r"""Hilbert space derived from AbstractParticle for
+    Fermions."""
+
+    def __init__(
+        self,
+        N: int,
+        L: Tuple[float, ...],
+        pbc: Union[bool, Tuple[bool, ...]],
+        mode=AnyStr,
+    ):
+        """
+        Constructs new ``Particles`` given specifications
+         of the continuous space they are defined in.
+        Args:
+            N: Number of particles
+            L: spatial extension in each spatial dimension
+            pbc: Whether or not to use periodic boundary
+                conditions for each spatial dimension.
+                If bool, its value will be used for all spatial
+                dimensions.
+        """
+        self._N = N
+        self._L = L
+        self._mode = mode
+
+        if isinstance(pbc, bool):
+            pbc = [pbc] * len(self._L)
+
+        self._pbc = pbc
+        if not len(self._L) == len(self._pbc):
+            raise ValueError(
+                """`pbc` must be either a bool or a tuple indicating the periodicity of each spatial dimension."""
+            )
+
+        super().__init__(self._L)
+
+    @property
+    def mode(self) -> AnyStr:
```

Comment:
do we actually need this now? can we avoid adding it until we will have a use case for it?

Reply:
yes I would remove it